mxnet-bot/LabelBotFullFunctionality/secret_manager.py

```diff
@@ -23,11 +23,12 @@
 import logging
 logging.basicConfig(level=logging.INFO)
 
+
 def get_secret():
-    '''
+    """
     This method is to fetch secret values
     Please configure secret_name and region_name as environment variables
-    '''
+    """
```

Comment:
Why make this change?

Reply:
Docstrings in python usually follow this notation (so this prevents any python lint errors)